.github/workflows/deploy.yml

```diff
@@ -94,6 +101,24 @@ jobs:
           file: "fly.toml"
           field: "app"
 
+      - name: Deploy PR
+        id: pr-deploy
+        uses: superfly/fly-pr-review-apps@1.0.0
+        if: ${{ github.event_name == 'pull_request' && github.event.label.name == 'staging' }}
```

Comment:
This might not actually work due to the event name piece. Not sure

Reply:
I'll add the label in a moment and give it a shot!

Also, I safe-listed the `superfly/fly-pr-review-apps` action in our repo settings.